src/App.tsx

```diff
@@ -1,56 +1,62 @@
-import React from 'react';
+import React, { useState, useCallback } from 'react';
+import { debounce } from 'lodash';
```

Comment:
You can install and use only debounce instead of all that have "lodash". https://www.npmjs.com/package/lodash.debounce